python/paddle/fluid/layers/nn.py

```diff
@@ -4253,62 +4253,69 @@ def layer_norm(input,
                act=None,
                name=None):
     """
-    ${comment}
+    **Layer Normalization Layer**
+
+    The OP implements the function of the Layer Normalization Layer and can be applied to mini-batch input data.
+    Refer to `Layer Normalization <https://arxiv.org/pdf/1607.06450v1.pdf>`_
 
     The formula is as follows:
 
     ..  math::
 
-        \\mu & = \\frac{1}{H}\\sum_{i=1}^{H} a_i
-
-        \\sigma & = \\sqrt{\\frac{1}{H}\sum_{i=1}^{H}(a_i - \\mu)^2}
-
-        h & = f(\\frac{g}{\\sigma}(a - \\mu) + b)
+        \\mu & = \\frac{1}{H}\\sum_{i=1}^{H} x_i
 
-    * :math:`a`: the vector representation of the summed inputs to the neurons
-    in that layer.
+        \\sigma & = \\sqrt{\\frac{1}{H}\sum_{i=1}^{H}{(x_i - \\mu)^2} + \\epsilon}
 
-    * :math:`H`: the number of hidden units in a layers
+        y & = f(\\frac{g}{\\sigma}(x - \\mu) + b)
 
-    * :math:`g`: the trainable scale parameter.
-
-    * :math:`b`: the trainable bias parameter.
+    - :math:`x`: the vector representation of the summed inputs to the neurons in that layer.
+    - :math:`H`: the number of hidden units in a layers
+    - :math:`\\epsilon`: the small value added to the variance to prevent division by zero.
+    - :math:`g`: the trainable scale parameter.
+    - :math:`b`: the trainable bias parameter.
 
     Args:
-        input(Variable): The input tensor variable.
-        scale(bool): Whether to learn the adaptive gain :math:`g` after
+        input(Variable): A multidimensional ``Tensor`` of any dimension, and the data type is float32 or float64.
```

Comment:
multi-dimension

Reply:
this sentence doesn't make sense, what do u mean about "A multidimensional ``Tensor`` of any dimension"

Reply:
done